1-js/04-object-basics/04-object-methods/article.md

```diff
@@ -259,6 +259,16 @@ That's a special feature of arrow functions, it's useful when we actually do not
 ## Summary
 
 - Functions that are stored in object properties are called "methods".
+- Method shorthand can be used instead of function expression:
+  
+  const User {
```

Comment:
It seems this needs to be:
`const user = {`
here and below (missing `=` and lowercased variable as this is not class name).
Also it seems the code fragments lack backticks.